### ISSUE TITLE
How to prepare FLAME_uv_template.obj and BFM_aligned.obj

### ISSUE THREAD

**Issue by pcgreat** [TimoBolkart/BFM_to_FLAME#18]
Nice work! Can you give a general introduction on how to prepare **FLAME_uv_template.obj** and **BFM_aligned.obj**?

Did you just prepare the generic template mesh of FLAME and save it as FLAME_uv_template.obj, and then align a template mesh of BFM to it via rigid transform or IPC?

**Comment by TimoBolkart** [comment]
Yes, that is precisely what we did. The BFM template was deformed with some non-rigid ICP to fit the FLAME template.

**Comment by pcgreat** [comment]
Thanks for the prompt reply! That's very helpful. Btw, did you happen to convert BFM textures to ICT (https://github.com/ICT-VGL/ICT-FaceKit) ?

**Comment by xiaolin-mt** [comment]
Could you please share the files, **FLAME_uv_template.obj** and **BFM_aligned.obj** ? @TimoBolkart

**Comment by TimoBolkart** [comment]
@pcgreat Neither BFM nor ICT are our models, so we cannot provide anything to convert between these models. What we do have and what we could share is something to convert from ICT to FLAME

**Comment by TimoBolkart** [comment]
@xiaolin-mt I am not sure if we are allowed to share BFM meshes. I will check

**Comment by xiaolin-mt** [comment]
> @xiaolin-mt I am not sure if we are allowed to share BFM meshes. I will check

Hi, I have checked the license of BFM. It prohibits the distribution to[ any third party](https://faces.dmi.unibas.ch/bfm/index.php?nav=1-2&id=downloads). So I assume you cannot share BFM meshes here. 
Could you please consider to share the pre-processing  scripts of both BFM and Flame data? so that we can apply for BFM data by ourselves under the license and rerun the entirely process. 
This repo appears to be very popular and valuable to study texture mapping between different topologies. I really appreciate for your sharing!

**Comment by WolfHolo2333** [comment]
> Nice work! Can you give a general introduction on how to prepare **FLAME_uv_template.obj** and **BFM_aligned.obj**?
> 
> Did you just prepare the generic template mesh of FLAME and save it as FLAME_uv_template.obj, and then align a template mesh of BFM to it via rigid transform or IPC?
May I ask if you have resolved this issue? I have recently had a request to use FaceKit, but it does not disclose texture bases

**Comment by sylyt62** [comment]
@TimoBolkart Could you share which tool did you use to perform non-rigid ICP? Matlab or any python package? Many thanks!

**Comment by TimoBolkart** [comment]
We used some internal Python-based registration code, that first fitted FLAME to the BFM mesh, followed by non-profit ICP. The scan fitting of [flame-fitting](https://github.com/Rubikplayer/flame-fitting) is a simplified version of what we used.
